remix-run/remix
# feat(remix-dev/compiler): add CSS plugin to `esbuild`

Closes: #1153


Testing Strategy:
I've tested it like:

- create demo remix app
- add css file with css assets (such as .svgs, .pngs, etc.)
- run remix in watch or build mode
- verify whether all assets are in the `_assets` folder (server and client)
- verify imported css paths are correct on server and client


Note: I have only tested relative url imports (`background: url('./relative');`), the following are untested:
- absolute url imports `background: url('/absolute');`
- data imports `background: url('data:image/gif;base64,...');`
- url imports `background: url('http://...');`
- idimports `background: url('#myid');`
